### ISSUE TITLE
iCloud sync support on iMac

### ISSUE THREAD

**Issue by kaihansen2001** [prof18/feed-flow#342]
Hello!!! 

Are there any plans to bring iCloud support to the iMac version??

Thanks in Advance!!!

**Comment by prof18** [comment]
Yes, I'm trying to understand how to integrate it. It could be possible, but I'm not promising anything. Stay connected :)
